### ISSUE TITLE
`default` response turned into `DEFAULT`

### ISSUE THREAD

**Issue by letmaik** [fastify/fastify-swagger#441]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure it has not already been reported

### Fastify version

3.19.1

### Plugin version

4.8.3

### Node.js version

15.14

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

18.04

### Description

When including `default` as one of the response schemas, then the generated swagger has `DEFAULT` as response key. 

### Steps to Reproduce

```
  app.get(
    '/foo', {
    schema: {
      response: {
        200: MySchema,
        default: ErrorSchema
      }
    }
  }, async (req, res) => {
...
```

### Expected Behavior

`default` should stay `default`.

**Comment by mcollina** [comment]
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by letmaik** [comment]
> Would you like to send a Pull Request to address this issue? Remember to add unit tests.

Possibly, do you have an idea which part of the code (or a dependency) would cause this issue?

**Comment by mcollina** [comment]
I have no clue. I didn't even know we supported `default`.

**Comment by letmaik** [comment]
You're right, fastify doesn't actually support what I'm trying to do. My end goal is to add a default response in the swagger doc for every route which seems quite common in order to deal with error responses.

**Comment by mcollina** [comment]
A PR to handle this case would be good!
